### ISSUE TITLE
I can't print the prediction in eval.py?

### ISSUE THREAD

**Issue by clgocn** [xiaodiu2010/TextBoxes-TensorFlow#16]
I wan't to print the predicitions values in the eval.py, but the result do not show。
`		with slim.arg_scope(arg_scope):
			localisations, logits, end_points  = \
				net.net(b_image, is_training=False, use_batch=FLAGS.use_batch)
		predictions = []
		for i in range(len(logits)):
			predictions.append(slim.softmax(logits[i]))
                        print(predictions)`
